pykeepass/pykeepass.py

```diff
@@ -45,6 +45,7 @@ class PyKeePass():
             database is assumed to have no keyfile
         transformed_key (:obj:`bytes`, optional): precomputed transformed
             key.
+        decrypt (:obj:`bool`, optional): whether to decrypt payload or not.
```

Comment:
I think this needs more explanation, its not clear what it does from an outsider perspective.

Reply:
Ok, then please help me out with a better description

Reply:
Ping :)

Reply:
Maybe something like 

    whether to decrypt XML payload.  Set `False` to access outer header information without decrypting database.

Reply:
Thanks for jumping in, i'll take it.